src/site/_data/paths/testing.json

```diff
@@ -0,0 +1,16 @@
+{
+    "slug": "testing",
+    "cover": "image/placeholder.jpg",
```

Comment:
I don't know how to add images as an external contributor, so I will use placeholders for now. How is the process in my case?